pages/Teams/Player_Profiles/player-profile-2.cfm

```diff
@@ -224,25 +224,32 @@
 	          </tbody>
 	        </table>
 		
-            <table class="bolder">
+             <cfif getPlayerAwards.recordCount EQ 0>
+              <!--- No data available, hide the table --->
+              <cfset tableStyle = 'display:none;'>
+            <cfelse>
+              <!--- Data available, show the table --->
+              <cfset tableStyle = 'display:table;'>
+            </cfif>
+
+            <table class="bolder" style="#tableStyle#">
               <caption>Awards</caption>
               <thead>
                 <tr class="headers">
-                    <th>Season</th>
-                    <th>Awards</th>
+                  <th>Season</th>
+                  <th>Awards</th>
                 </tr>
               </thead>
               <tbody>
                 <cfloop query="getPlayerAwards">
-                    <tr>
-                        <td data-label="Season">#getPlayerAwards.SeasonName#</td>
-                        <td data-label="Awards">#getPlayerAwards.AwardName#</td>
-                    </tr>
+                  <tr>
+                    <td data-label="Season">#getPlayerAwards.SeasonName#</td>
+                    <td data-label="Awards">#getPlayerAwards.AwardName#</td>
+                  </tr>
                 </cfloop>
               </tbody>
-            </table>        
```

Comment:
You can take the logic you had above and apply it the table itself instead of having to set a style for it. So something like. This is called wrapping the table in an IF condition.
```
<cfif getPlayerAwards.recordCount NEQ 0>
 <table class="bolder">
              <caption>Awards</caption>
              <thead>
                <tr class="headers">
                    <th>Season</th>
                    <th>Awards</th>
                  <th>Season</th>
                  <th>Awards</th>
                </tr>
              </thead>
              <tbody>
                <cfloop query="getPlayerAwards">
                    <tr>
                        <td data-label="Season">#getPlayerAwards.SeasonName#</td>
                        <td data-label="Awards">#getPlayerAwards.AwardName#</td>
                    </tr>
                  <tr>
                    <td data-label="Season">#getPlayerAwards.SeasonName#</td>
                    <td data-label="Awards">#getPlayerAwards.AwardName#</td>
                  </tr>
                </cfloop>
              </tbody>
            </table>    
</cfif>
```

Where **NEQ** means Not equal to 0.